### ISSUE TITLE
Error "Cannot decode JSON from PhantomJS runner" for phantomjs v1.9.8 (2)

### ISSUE THREAD

**Issue by dima4p** [guard/guard-jasmine#186]
Hello, I get this message and the reason is that phantomj returns multi-line result like this
```
$ ~/.phantomjs/1.9.8/x86_64-linux/bin/phantomjs /usr/local/rvm/gems/ruby-2.2.1@pe_voip_server/gems/guard-jasmine-2.0.5/lib/guard/jasmine/phantomjs/guard-jasmine.js "http://localhost:8888/?spec=Vox" 60000
{"error":"Javascript error encountered on Jasmine test page: ReferenceError: Can't find variable: module","trace":[{"file":"http://localhost:8888/__spec__/helpers/vox_mock.js","line":14,"function":""}]}
{"error":"Javascript error encountered on Jasmine test page: ReferenceError: Can't find variable: require","trace":[{"file":"http://localhost:8888/__spec__/session_spec.js","line":10,"function":""}]}
{"error":"Javascript error encountered on Jasmine test page: ReferenceError: Can't find variable: require","trace":[{"file":"http://localhost:8888/__spec__/vox_spec.js","line":7,"function":""}]}
```
So, when JSON parser meets the second line in :evaluate_response method it raises an error.
I'm not sure if it is possible to break the string json into several lines and process them one by one.

**Comment by nathanstitt** [comment]
Thanks for reporting the issue @dima4p.  I believe the error you encountered traces back to your page having multiple javascript errors present, and then GuardJasmine attempted to log them all individually.  I've changed how they're reported so it will only log them one at a time.

I just released version 2.0.6 with the changes, please give it a spin and see if it works for you.

**Comment by nathanstitt** [comment]
I'm pruning old issues and hope the new version worked for you, please open a new issue if you're still having problems
